testdata/p4_16_samples/psa-unicast-or-drop-bmv2.stf

```diff
@@ -0,0 +1,8 @@
+# ingoing packet
```

Comment:
Can we use the unicast or dorp stf test from andy's repository? Not sure why it expects 4 packets when only one is fed into the switch.